Fix SG rules conflicting constraints doc

falzm opened the pull request:
This change adds missing attributes conflicting constraints and fixes
the documentation for the `exoscale_security_group_rule*` resources.

greut commented:
The following approach is probably best advised: https://rsmitty.github.io/Terraform-Ansible-Kubernetes/

Commit:
Fix SG rules conflicting constraints doc

This change adds missing attributes conflicting constraints and fixes
the documentation for the `exoscale_security_group_rule*` resources.
